pentaho/src/test/java/pt/webdetails/cpf/olap/AbstractOlapUtilsTest.java

```diff
@@ -125,12 +120,10 @@ public void testGetMeasures() throws Exception {
         "\"caption\":\"member with type regular\",\"type\":\"measure\"},{\"qualifiedName\":\"[Unknown]\"," +
         "\"name\":\"unknown\",\"memberType\":\"UNKNOWN\",\"caption\":\"member with type unknown\"," +
         "\"type\":\"measure\"}]";
-    try {
-      result = olapUtils.getMeasures( mockedConnection, "" );
-    } catch ( JSONException e ) {
-      Assert.fail( e.getMessage() );
-    }
-    Assert.assertTrue( jsonEquals( expectedResult, result.toString() ) );
+
+    result = olapUtils.getMeasures( mockedConnection, "" );
+
+    assertTrue( jsonEquals( expectedResult, result.toString() ) );
```

Comment:
Can you use here the assertEquals?

Reply:
No, because the comparison involves some intermediate transformations.